src/Security/Authentication/Cookies/src/CookieAuthenticationEvents.cs

```diff
@@ -18,6 +18,11 @@ public class CookieAuthenticationEvents
         /// </summary>
         public Func<CookieValidatePrincipalContext, Task> OnValidatePrincipal { get; set; } = context => Task.CompletedTask;
 
+        /// <summary>
+        /// Invoked to check if the cookie should be renewed.
+        /// </summary>
+        public Func<CookieSlidingExpirationContext, Task>? OnCheckSlidingExpiration { get; set; }
```

Comment:
@davidfowl is it worth the inconsistency?

Reply:
Nah, leave it consistent.